lib/src/view/study/study_screen.dart

```diff
@@ -568,6 +569,7 @@ class _StudyBoardState extends ConsumerState<_StudyBoard> {
                 },
               )
               : null,
+      boardPrefs: boardPrefs,
```

Comment:
nit: same here, maybe we add this new line right under the size argument to keep argument order the same